src/core/__tests__/GoogleCalendar.spec.js

```diff
@@ -0,0 +1,119 @@
+import CalendarBase from '../CalendarBase'
+import GoogleCalendar from '../GoogleCalendar'
+
+import { FORMAT, RECURRENCE, URL } from '../../constants'
+import { formatTime } from '../../utils/time'
+import * as Ics from '../../utils/ics'
+// import { getRrule } from '../../utils/ics'
+jest.mock('../../utils/ics')
+
+import queryStringToObj from '../../../test_helpers/queryStringToObj'
+
+const { FREQUENCY: { DAILY } } = RECURRENCE
+describe('GoogleCalendar', () => {
+  const baseOpts = {
+    title: 'Test Event',
+    description: 'Test Description',
+    location: 'Rockefeller Center',
+    start: '20150704T190000'
+  }
+  const baseParams = {
+    text: 'Test Event',
+    details: 'Test Description',
+    location: 'Rockefeller Center',
+    action: 'TEMPLATE'
+  }
+
+  let testObj
+  afterEach(() => {
+    testObj = undefined
+    jest.resetAllMocks()
+  })
+
+  it('should be a subclass of CalendarBase', () => {
+    expect(new GoogleCalendar({})).toBeInstanceOf(CalendarBase)
+  })
+
+  describe('render()', () => {
+    let result;
+
+    afterEach(() => {
+      result = undefined
+    })
+
+    it('should use the proper base URL', () => {
+      testObj = new GoogleCalendar(baseOpts)
+
+      result = testObj.render()
+      let baseUrl = result.split('?')[0]
+
+      expect(baseUrl).toBe(URL.GOOGLE)
+    })
+
+    it('should handle the text parts', () => {
+      testObj = new GoogleCalendar(baseOpts)
+
+      result = testObj.render()
+      let queryString = result.split('?')[1]
+      let paramsObj = queryStringToObj(queryString)
+
+      expect(paramsObj).toMatchObject(baseParams)
+    })
+
+    describe('an all-day event', () => {
+
+      it('should have start and end date formatted without time-of-day', () => {
+        testObj = new GoogleCalendar(baseOpts)
+
+        result = testObj.render()
+        let queryString = result.split('?')[1]
+        let paramsObj = queryStringToObj(queryString)
+
+        let expectedDates = `${
+          formatTime(testObj.start, FORMAT.DATE)
+        }/${
+          formatTime(testObj.end, FORMAT.DATE)
+        }`
+
+        expect(paramsObj.dates).toBe(expectedDates)
+      })
+
+    })
+
+    describe('without recurrence', () => {
+
```

Comment:
please fix the spacing between the `describe`s and `it`s so that it's consistent and readable

```js
describe('the thing', () => {
  beforeEach(() => {
    // do stuff
  })

  it('should ...', () => {
  })

  it('should ...', () => {
  })
})